src/utils/events/constants.js

```diff
@@ -80,7 +80,7 @@ export const RECOGNIZERS = [
   [Swipe, {enable: false}],
   [Press, {enable: false}],
   [Tap, {event: 'doubletap', taps: 2, enable: false}],
-  [Tap, {enable: false, interval: 0}]
+  [Tap, {enable: false, interval: 0}, null, 'doubletap']
```

Comment:
Unless this is obvious to any Hammer.js user, perhaps add a comment here about what this does (the same comment you made in this PR)?